src/librustc_data_structures/bitvec.rs

```diff
@@ -293,6 +293,11 @@ impl<R: Idx, C: Idx> SparseBitMatrix<R, C> {
     ///
     /// Returns true if this changed the matrix, and false otherwise.
     pub fn add(&mut self, row: R, column: C) -> bool {
+        self.vector.resize_to_elem(row, || SparseBitSet::new());
+        if let None = self.vector.get(row) {
```

Comment:
I think this `if let` is dead-code now, right?

Reply:
Yes, must have forgot to clean that up. I'll remove this in my next commit.